### PR TITLE
Modifying documentation on Configuring Environment Specific Parameters

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Fixes: https://github.com/wso2/product-apim-tooling/issues/270

## Goals
Move the content on **Configuring Environment Specific Parameters** under **Advanced Topics**.

## Approach
- Created a new page titled **Configuring Environment Specific Parameters** under **Advanced Topics** as shown below.
image
- Removed the repetitive content related to the above topic from other pages and referred to this page using Tips.
  - Removed the section named "Configure Environment Specific Details" from the page **Importing APIs Via Dev First Approach**. Updated the reference within the content in the flags to point to the newly created page as shown below.
image
  - Removed the section named "Configuring Environment Specific Parameters" from the page **Migrating APIs to Different Environments** and referred to the new page within a Note.
image
  - Updated the hyperlink named "Configuring Environment Specific Parameters" on the page **CI/CD with WSO2 API Manager**.
image
  - Updated the hyperlink named "Configuring Environment Specific Parameters" on the page **Building a Jenkins CI/CD Pipeline for Dev First Approach**.
image

## User stories
Users can refer to the newly added page to learn on using api_params.yaml file to configure environment specific details.

### CODE REVIEW

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+           - enabled: true
+             type: digest
+             username: admin
+             password: admin
+              config:
+                retryTimeOut: $RETRY
+            sandbox:
+              url: 'https://test.sandbox.wso2.com'
+    ```
+Instead of the default `api_params.yaml`, you can a provide custom parameter file using `--params` flag. A sample command will be as follows.
```
Comment (hisanhunais) [inline en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:71]: Shall we change this to "you can provide a"?

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+!!! example
+    ```go
+    apictl import-api -f dev/PhoneVerification_1.0.zip -e production --params /home/user/custom_params.yaml 
+    ```
+
+!!! info
+    -   Production/Sandbox backends for each environment can be specified in the parameter file with additional configurations, such as timeouts.
+    -   Under the security field, if the `enabled` attribute is `true`, you must specify the `username`, `password` and the `type` (can be either only `basic` or `digest`). If the `enabled` attribute is `false`, then non of the security parameters will be set. If the `enabled` attribute is not set (blank), then the security parameters in api.yaml file will be considered.
```
Comment (hisanhunais) [inline en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:80]: In the middle of the second line, shall we change "non" to "none"?

Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:80]: This is already fixed in the master branch via https://github.com/wso2/docs-apim/commit/c049f0216cc3f99dbd8b9cfd5d3021c318f365d2
Hence we can ignore this.

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md (hunk 1 above)
Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:71]: Fixed via https://github.com/wso2/docs-apim/pull/3097/commits/74cdaa1323d12a99abd038f41a690914b8d50ab5

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md (hunk 2 above)
Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:80]: Fixed via https://github.com/wso2/docs-apim/pull/3098/commits/edd8ae4c020155b3c9d8e4465a923a031e8f9472 for 3.2.0

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md (hunk 1 above)
Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:71]: Fixed via https://github.com/wso2/docs-apim/pull/3098/commits/edd8ae4c020155b3c9d8e4465a923a031e8f9472 for 3.2.0

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md (hunk 2 above)
Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:80]: Fixed via https://github.com/wso2/docs-apim/pull/3099/commits/5f5c672fdf5973c1368873c40f65a138e2fbe58a for 3.1.0

File: en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md (hunk 1 above)
Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:71]: Fixed via https://github.com/wso2/docs-apim/pull/3099/commits/5f5c672fdf5973c1368873c40f65a138e2fbe58a for 3.1.0

Comment (wasuradananjith) [reply en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md:71]: Fixed via https://github.com/wso2/docs-apim/pull/3100/commits/0dde1cd55e1981bc950fada7f2b96ba5263d5deb for 3.0.0

**Pull request merged**